packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -377,6 +377,10 @@ const Popup: React.FC<PopupProps> &
   };
 
   const shouldBlurClose = e => {
+    console.log('shouldBlurClose', e);
```

Comment:
Please could you remove console.log before merging?

Reply:
Cruft, good catch